packages/server/src/routes/api/users/users-router.js

```diff
@@ -59,7 +64,22 @@ router.post(
   async (req, res, next) => {
     try {
       const validationResults = await validateEmailAndUsername(req.body);
-      res.status(200).send({ ...validationResults, emailExists: false });
+      res.status(200).send({ ...validationResults });
+    } catch (err) {
+      next(err);
+    }
+  },
+);
+
+router.patch(
+  "/:id",
+  [emailAndUsernameUpdateValidationFn],
+  async (req, res, next) => {
+    try {
+      // TODO: validate email/user for update (ensure neither have been taken)
```

Comment:
Should be fine since the db would throw if either are taken

Reply:
rn I have @danyliu9 's changes from his pr merged in so the user update comments are for https://github.com/jpfranci/Alpacow/pull/26 except for one